cmd/docker/docker.go

```diff
@@ -48,11 +41,12 @@ func newDockerCommand(dockerCli *command.DockerCli) *cobra.Command {
 			}
 			return isSupported(cmd, dockerCli)
 		},
+		Version: fmt.Sprintf("%s, build %s", cli.Version, cli.GitCommit),
 	}
 	cli.SetupRootCommand(cmd)
 
 	flags = cmd.Flags()
-	flags.BoolVarP(&opts.Version, "version", "v", false, "Print version information and quit")
+	flags.BoolP("version", "v", false, "Print version information and quit")
```

Comment:
Unfortunately still needed; our flag-description has a short-hand (`-v`), and looks like;

```
 -v, --version            Print version information and quit
```

Whereas the built-in flag for Cobra doesn't have a shorthand, and looks like this;

```
      --version            version for docker
```